### ISSUE TITLE
Drag & Drop - by mouse - to Recycle Bin

### ISSUE THREAD

**Issue by JoxCoswit** [tablacus/TablacusExplorer#501]
Dear Gaku, 

Is it possible to modifay **DELETE** [**ADD-ON**] in settings (**Add-ons>Delete>Options:** **Menus**/**Key**/**Mouse**) to accept and enable function **DRAG** & **DROP** for delete content?

![TE](https://user-images.githubusercontent.com/60333254/145277327-15d2b57a-0b21-4867-804e-518d42e0441f.jpg)
**Description:** 
I tried to move selected folder to icon Recycle Bin (ADD-ON: Delete), I got was icon of denied.
In other words... not available, or not allowed.

**Descriptive functions of the delete button:**
- **1st option**: Select content and by pressing delete button (icon: recycle bin) -- content has been deleted (as it is right now)
- **2nd option**: Take and drag by the mouse onto icon "recycle bin" (as it exists in Windows) -- to be deleted.

I was looking through ADD-ONS and I haven't found any on the list that would have (or with) similar functionality for deleting like that "DRAG & DROP"

![Windows](https://user-images.githubusercontent.com/60333254/145276242-c9c39708-772a-42ae-8de4-ccfa974c9356.jpg)

Best regards,

**Comment by tablacus** [comment]
I've added it with the new feature of the add-on Delete.
Give it a try.

Regards,

**Comment by JoxCoswit** [comment]
Sorry, but it didn't work.
Ver of add-on **Delete 1.06** (03.04.2021)...?

I try to update... there is noting to update.
Then I try next thing:

- I download brand new folder of TE "te211208.zip"
- Install "Delete" (I take screenshot what I actually see) 
,,,,and nothing happend (icon is still red with sign of denied)

Is this maybe old version of add-on Delete?
![Delert2](https://user-images.githubusercontent.com/60333254/145503141-3e77070d-123a-4a9a-acf1-a014df7f1503.jpg)


Best regards.

**Comment by tablacus** [comment]
I'm sorry, I made a mistake in uploading.

Regards,

**Comment by JoxCoswit** [comment]
No problem, I just updated.
Now, that is another story.... perfect....It works as I imagined  😀
This way of deleting is very helpful for me.
Anymore I don't have to carry all the way to the recycle bin which is located on desktop.
Or be afraid that I will delete something if I have previously marked incorrectly.
Now I can just grab and move to the recycle bin to be deleted.

Thank you Gaku for that!  👍

Best regards.
